services/QuillLMS/app/views/pages/grammar_tool.erb

```diff
@@ -1,146 +1,70 @@
-    </p>
-    <h4 class="q-h4 author">
-      <span>Daniel Scibienski</span>, Princeton Public Schools
-    </h4>
-    <a href="/activities/packs/6" class='q-button text-black'>
-      <span>View My Story</span>
-    </a>
+  <%= render partial: "pages/shared/research_blocks", locals: { research_blocks: [
+    { icon: "lecture-hall-purple.svg", title: "Grammar Practice", description: "Students are tasked with writing a complete sentence by incorporating proper grammar, punctuation, and spelling. Students then receive instant feedback on the quality of their writing." },
+    { icon: "teacher-whiteboard-purple.svg", title: "Instant Feedback", description: "Students are provided with an opportunity to try again if they make a mistake. If, on the second attempt, a student does not write a complete sentence, the program shows the student where the student’s error is." },
+    { icon: "fountain-pen-purple.svg", title: "In-Depth Teacher Reports", description: "Teachers access in-depth reports that show them how the students have performed on all of the materials over the course of the year." },
+    { icon: "student-study-purple.svg", title: "Common Core Language Standards", description: "Teachers have access to over 200 exercises aligned with the Common Core language standards." }
+  ] } %>
+  <%= render partial: "pages/shared/targeted_skills", locals: {
+    heading: "What Skills Are Being Targeted",
+    tags: [
+      "Adjectives",
+      "Articles",
+      "Capitalization",
+      "Commonly Confused Words",
+      "Contractions",
+      "Nouns",
+      "Prepositions",
+      "Pronouns",
+      "Puncuation",
+      "Questions",
+      "Relative Clauses",
+      "Simple Sentence Structure",
+      "Verbs"
+    ],
+    buttons: [
+      { text: "Browse 150+ Activities", href: "/activities/packs?type=grammar", color: "purple" }
+    ]
+  } %>
+  <%= render partial: "pages/shared/relevant_articles", locals: { articles: @featured_grammar_articles, color: "purple" } %>
+  <section class="teacher-quote-container">
+    <p>"I've seen tremendous improvement in the proficiency of my students and the quality of their writing."</p>
+    <p>Daniel Scibienski, Princeton Public Schools</p>
   </section>
-
-
-  <!-- Only show if user is not logged in  -->
-    <% if !current_user %>
-      <section class="tool-sign-up">
-          <h1 class='q-h1'>
-              Sign up to start Quill Grammar in your class today!
-          </h1>
-          <a href='/account/new' class="q-button cta-button bg-quillgreen text-white">Sign Up</a>
-      </section>
-
-    <% else %>
-    <section class="tool-sign-up">
-        <h1 class='q-h1'>
-            Get started now! Assign Quill Grammar.
-        </h1>
-        <a href='/assign/activity-library?activityClassificationFilters[]=sentence' class="q-button cta-button bg-quillgreen text-white">Assign</a>
-    </section>
-    <% end %>
-
-
+  <%= render partial: "pages/shared/assign_activity", locals: { tool: "passage" } %>
```

Comment:
I think Grammar's key is actually `sentence`. These keys are stored on the `ActivityClassification` model, maybe we should be referencing those here?

Reply:
yes, great call-- I'll pass the `ActivityClassification` object directly